iot-hub/Samples/device/MessageReceiveSample/MessageReceiveSample.cs

```diff
@@ -84,7 +88,9 @@ private async Task ReceiveC2dMessagesPollingAndComplete(CancellationToken ct)
                     break;
                 }
 
-                using Message receivedMessage = await _deviceClient.ReceiveAsync(s_sleepDuration);
+                using Message receivedMessage = await _transportType == TransportType.Http1
+                    ? _deviceClient.ReceiveAsync()
```

Comment:
await on the receive call, instead of for the transport type

Reply:
Either is legit. Either I have to write await twice, or I write it once.

Reply:
some braces missing then?
```csharp
using Message receivedMessage = await (_transportType == TransportType.Http1
                    ? _deviceClient.ReceiveAsync()
                    : _deviceClient.ReceiveAsync(s_sleepDuration));
```

Reply:
I have to eat my words here. It doesn't build. I'm sure I could mess with it a bit, but I won't bother. I've changed it as you've requested.